sessions/biases-in-delay-distributions.qmd

```diff
@@ -138,49 +162,58 @@ cres <- cmod$sample(
 cres
 ```
 
-::: {.callout-tip}
+::: callout-tip
 ## Take 10 minutes
+
 Try re-simulating the delays using different parameters of the delay distribution.
 Can you establish under which conditions the bias in estimation gets worse?
 :::
 
 ## Real-time estimation: truncation
 
 The data set we have looked at so far in this session is a "final" data set representing an outbreak that has come and gone.
-However, information e.g. on delay distribution is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.
-Estimating delays in real time comes with particular challenges as the timing of the cut-off might introduce a bias.
-If, for example, infections are exponentially increasing then there will be disproportionately more people with recent symptom onset, decreasing the estimate of the mean delay.
+However, information on delay distribution is often important during ongoing outbreaks as they can inform nowcasts and forecasts and help with broader interpretation of data.
+
+Estimating delays in real time comes with particular challenges, as the timing of the cut-off might introduce a bias.
+If, for example, infections are exponentially increasing then there will be disproportionately more people with recent symptom onset.
+This would artificially decrease the estimate of the mean delay compared to its true value for all infections.
```

Comment:
```suggestion
Without adjustment, this would artificially decrease the estimate of the mean delay compared to its true value for all infections.
```